src/plugins/plot/stackedPlot/mixins/objectStyles-mixin.js

```diff
@@ -128,36 +128,26 @@ export default {
       }
     },
     updateStyle(styleObj) {
-      let elemToStyle = this.getStyleReceiver();
+      const elemToStyle = this.getStyleReceiver();
 
-      if (!styleObj || elemToStyle === undefined) {
+      if (!styleObj || !elemToStyle) {
         return;
       }
 
-      let keys = Object.keys(styleObj);
+      // handle visibility separately
+      if (styleObj.isStyleInvisible !== undefined) {
+        elemToStyle.classList.toggle(STYLE_CONSTANTS.isStyleInvisible, styleObj.isStyleInvisible);
+        styleObj.isStyleInvisible = null;
+      }
 
-      keys.forEach((key) => {
-        if (elemToStyle) {
-          if (typeof styleObj[key] === 'string' && styleObj[key].indexOf('__no_value') > -1) {
-            if (elemToStyle.style[key]) {
-              elemToStyle.style[key] = '';
-            }
+      requestAnimationFrame(() => {
```

Comment:
`requestAnimationFrame` needs a semaphore, otherwise it does not throttle anything, the requestAnimationFrame calls will just queue up in the event loop. You can use https://github.com/nasa/openmct/blob/master/src/utils/raf.js#L1 for this.